stories/atoms/components/icon/icon.scss

```diff
@@ -14,6 +14,12 @@
   }
 }
 
+.invert {
```

Comment:
I think `inverted` is more suitable for the classname, just like below `inverted-field`.